app/form_models/publishers/job_listing/about_the_role_form.rb

```diff
@@ -9,6 +9,8 @@ class Publishers::JobListing::AboutTheRoleForm < Publishers::JobListing::Vacancy
   validate :safeguarding_information_does_not_exceed_maximum_words, if: -> { safeguarding_information_provided == "true" }, unless: -> { vacancy.job_advert.present? || vacancy.about_school.present? }
   validates :further_details_provided, inclusion: { in: [true, false, "true", "false"] }, unless: -> { vacancy.job_advert.present? || vacancy.about_school.present? }
   validate :further_details_presence, if: -> { further_details_provided == "true" }, unless: -> { vacancy.job_advert.present? || vacancy.about_school.present? }
+  validates :flexi_working_details_provided, inclusion: { in: [true, false, "true", "false"] }
```

Comment:
it would be more idiomatic to make this field an attribute of type boolean rather than a string. Then the checks aren't required, and the test becomes boolean rather than x == "true" (which as you allow 'true' is technically broken maybe?).

I don't think we need to persist the 'flexi working provided' value - check out TRN where IIRC I made the 'provided' value non-persisted, validates it based on true and loaded it based on data presence